repos/system_upgrade/common/actors/scansubscriptionmanagerinfo/tests/test_scansubscriptionmanagementinfo.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from leapp.libraries.actor import scanrhsm
+from leapp.libraries.common import rhsm
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api
+from leapp.models import RepositoryData, RepositoryFile, RHSMInfo
+
+
+def mocked_get_rhsm_info(context):
+    assert context, 'The actor did not provide library with valid context.'
+    info = RHSMInfo()
+    info.attached_skus = ['SKU1', 'SKU2']
+    info.available_repos = ['Repo1', 'Repo2']
+    info.enabled_repos = ['Repo2']
+    info.release = '7.9'
+    info.existing_product_certificates.extend(['Cert1', 'Cert2', 'Cert3'])
```

Comment:
why extend when you set what should be inside? The assert below expects excatly the same list. The extend doesn't not make sense in such a case. If there is specific reason for the extend, just let me know.

```suggestion
    info.existing_product_certificates = ['Cert1', 'Cert2', 'Cert3']
```

Reply:
Sorry :confused:, I don't get it why did I write it this way and there is no reason why do not use direct assignment.